content/docs/1.0.0/advanced-resources/deploy/airgap.md

```diff
@@ -8,7 +8,9 @@ weight: 2
   - Deploy Kuberntes CSI driver components images to your own registry.
 
 #### Note:
-  - CSI driver component's images, names and tags can be found [here.](../../../concepts/#13-csi-driver)
+  - A full list of all needed images is in [longhorn-images.txt](https://raw.githubusercontent.com/longhorn/longhorn/v1.0.1/deploy/longhorn-images.txt)
+  - We provide a script to quickly pull and export the above `longhorn-images.txt` list to a `tar` file: [save-images.sh](https://raw.githubusercontent.com/longhorn/longhorn/v1.0.1/scripts/save-images.sh)
+  - We provide another script to load images from the `tar` file and push them to your private registry: [load-images.sh](https://raw.githubusercontent.com/longhorn/longhorn/v1.0.1/scripts/load-images.sh)
```

Comment:
Need to have more details on how to use the scripts, including example here.

Reply:
added more info